azure-py-virtual-data-center/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi>=1.0.0
```

Comment:
These need to have an upperbound so it would become

```
pulumi>=1.0.0,<2.0.0
pulumi-azure>=2.0.0,<3.0.0
```

Without this we can get into a state of incompatible pulumi versions